utils/index.d.ts

```diff
@@ -2070,3 +2070,33 @@ export declare class TreeElementStateManager<TElement extends TreeElementWithId
 
     dispose(): void;
 }
+
+/**
+ * Input string validation utilities for Azure Tools
+ */
+export declare namespace validationUtils {
+    /**
+     *  Defines the constraints for a valid size range
+     *  @property `lowerLimitIncl`: The minimum size of the range (inclusive)
+     *  @property `upperLimitIncl`: The maximum size of the range (inclusive)
+     */
+    export interface RangeConstraints {
```

Comment:
Kept this named very generically because I was planning to reuse this interface again later for another validation method